### PR TITLE
Update Submission Payload

### PR DESCRIPTION
**Pull request by dimak1**
*Description of changes:*
- Update payload data for submission
- Confirm if Submitting Party is a Business Account info (as in UI/Requirements) or a Current User's info

#### Submitting Party on the Review Screen:

Screen Shot 2022-11-07 at 1 44 21 PM

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of the PPR license (Apache 2.0).

### CODE REVIEW

File: ppr-ui/src/composables/mhrInformation/useMhrInformation.ts [hunk 1]
```diff
@@ -85,22 +91,23 @@ export const useMhrInformation = () => {
       transferDate: getMhrTransferDate.value,
       ownLand: getMhrTransferOwnLand.value,
       attentionReference: getMhrTransferAttentionReference.value,
-      documentDescription: 'SALE OR GIFT',
+      documentDescription: UIRegistrationTypes.TRANSFER_OF_SALE,
       submittingParty: {
+        // Need to confirm if this needs to be a Business Name instead
+        // because in UI, Submitting Party is a business not a person
+        // but the API spec requires firstName, lastName
```
Comment (cameron-eyds) [inline ppr-ui/src/composables/mhrInformation/useMhrInformation.ts:98]: Submitting party can be a person OR a business afaik

File: ppr-ui/src/composables/mhrInformation/useMhrInformation.ts [hunk 2]
```diff
@@ -85,22 +91,23 @@ export const useMhrInformation = () => {
...
-        emailAddress: getCurrentUser.value.contacts[0].email,
-        phoneNumber: getCurrentUser.value.contacts[0].phone.replace(/[^0-9.]+/g, '') // Remove special chars
-      },
+        address: getMhrTransferSubmittingParty.value.address,
+        emailAddress: getMhrTransferSubmittingParty.value.emailAddress,
+        phoneNumber: getMhrTransferSubmittingParty.value.phoneNumber,
+        phoneExtension: getMhrTransferSubmittingParty.value.phoneExtension
+        // Cleanup below after confirming that Submitting Party info is Account's Admin info
+        // emailAddress: getCurrentUser.value.contacts[0].email,
+        // phoneNumber: getCurrentUser.value.contacts[0].phone.replace(/[^0-9.]+/g, '') // Remove special chars
```
Comment (cameron-eyds) [inline ppr-ui/src/composables/mhrInformation/useMhrInformation.ts:109]: I think you can confirm with Tracey and remove these comments 👍 
That being said, there is no submitting party information completed in a Transfer filing so how are the root properties of `getMhrTransferSubmittingParty` populated? 
Is it current account information?

Comment (cameron-eyds) [reply ppr-ui/src/composables/mhrInformation/useMhrInformation.ts:109]: Ah i looked below and see. 
So isn't saving the results of `getAccountInfoFromAuth` ie currentAccount to both current account info and submitting party info a bit of a decoy? ie storing the same information twice under two different alias.

*Sorry another edit, not saying anything is wrong here, just that we have a lot of these decoys/patterns to be aware of :(

Comment (cameron-eyds) [reply ppr-ui/src/composables/mhrInformation/useMhrInformation.ts:109]: Or is that different from current account?

**Comment by dimak1** [comment]
@cameron-eyds updated the Description with a screenshot from UI requirements. We do display the Account name. In the database, the name is just one field. So currently (in our submit call) we pass the Current User's first name and last name, which I think should be an Account name. I also believe the current API spec is outdated, and Thor will send me the latest specs.

File: ppr-ui/src/composables/mhrInformation/useMhrInformation.ts (hunk 2 above)
Comment (dimak1) [reply ppr-ui/src/composables/mhrInformation/useMhrInformation.ts:109]: Current account is the logged-in user (e.g BCREG2 Della FORTYTHREE). The Account Info contains the account name (e.g. PPR 2). So they are not the same. We are making a separate call to `/orgs` to get name and address info. We are doing some parsing to align the prop names of the interfaces. We could probably optimize it still.

Comment (cameron-eyds) [reply ppr-ui/src/composables/mhrInformation/useMhrInformation.ts:109]: For sure, the `/orgs` request is part of `getAccountInfoFromAuth` right?
That is called in `MhrInformation` via `getSubmittingPartyInformation`, saved to localState and subsequently submitting party info. So what do we designate that? 
It's misleading as the CONST used in that request is `CurrentAccount`

Comment (cameron-eyds) [reply ppr-ui/src/composables/mhrInformation/useMhrInformation.ts:109]: So to be plain, Submitting Party is or is not the Current Account?

Comment (dimak1) [reply ppr-ui/src/composables/mhrInformation/useMhrInformation.ts:109]: Just to reply to your comment. Based on **currentAccount** id we make a calls to `/orgs` to get Account Info submitting party.

**Comment by dimak1** [comment]
This is an old PR and has many conflicts, and rebasing would take a long time. Opened a new PR #1131 instead. Will close this soon.

**Pull request closed without merging**